src/storage/src/hummock/state_store.rs

```diff
@@ -362,6 +331,45 @@ impl HummockStorage {
         Ok(None)
     }
 
+    async fn find_value_from_order_sorted_uncommitted_data(
```

Comment:
Name it `get_from_order_sorted_uncommitted_data` to be consistent with other method. Please also document the meaning of the return type.